test/Interop/Cxx/class/invalid-class-errors.swift

```diff
@@ -0,0 +1,38 @@
+// RUN: rm -rf %t
+// RUN: split-file %s %t
+// RUN: not %target-swift-frontend -typecheck -I %t/Inputs  %t/test.swift  -enable-experimental-cxx-interop 2>&1 | %FileCheck %s
+
+//--- Inputs/module.modulemap
+module Test {
+    header "test.h"
+    requires cplusplus
+}
+
+//--- Inputs/test.h
+struct X;
+
+struct A {
+        A(const A&) = delete;
+};
+
+struct __attribute__((swift_attr("import_unsafe"))) B {
+    B(const B&) = delete;
+};
+
+struct C {
+        C(const C&) {}
+};
+
+//--- test.swift
+
+import Test
+
+// CHECK: note: record 'X' is not defined (incomplete)
```

Comment:
@NuriAmari thanks again for building all the infrastructure for these diagnostics. It made it super easy (one line) to add diagnostics for all these other cases.

Reply:
My pleasure, glad it's being put to good use 👍